src/ucp/core/ucp_worker.c

```diff
@@ -2455,10 +2474,9 @@ ucs_status_t ucp_worker_query(ucp_worker_h worker,
             }
         }
 
-        status = ucp_address_pack(worker, NULL, &tl_bitmap,
-                                  ucp_worker_default_address_pack_flags(worker),
-                                  NULL, &attr->address_length,
-                                  (void**)&attr->address);
+        status = ucp_worker_address_pack(worker, &tl_bitmap,
```

Comment:
move tl_bitmap generation inside ucp_worker_address_pack and address_flags would be a parameter for ucp_worker_address_pack 
reason: encapsulate all worker-address-packing related logic inside ucp_worker_address_pack

Reply:
maybe in a separate PR, as it is unrelated change?